src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/main.js

```diff
@@ -918,7 +918,7 @@ define([
                 // thats currently not supported by the api
                 var cur = element.parentNode;
                 while (null !== cur.parentNode) {
-                    if (cur.hasAttribute('property')) {
+                    if (cur.hasAttribute('property') && cur.getAttribute('typeof') === 'collection') {
```

Comment:
when i find a block here i dont want to use this property! thats why i ?`return false` in the next line!